### PR TITLE
Feature/styling2

### LINKED ISSUE #34: Adjust styling on Attributes box

### LINKED ISSUE #35: Add Treezy logo to error page

### LINKED ISSUE #36: Change icon and title of chrome tab for app

### LINKED ISSUE #37: Edit Error component's styling

### PR DESCRIPTION
**Pull request by irmakerdem**
What does this PR do?
- Adds styling and logo to Error component
- Adds styling to Attributes box
- Adds tree emoji and app title to browser's tab
- Adjusts background opacity to be even

Where should your reviewer start?
- Line 75 in README.ms
- Line 5 in index.html
- Line 7 in DetailContainer.css
- Line 1 in Error.css
- Line 2 in Error.js
- Line 28 in Home.css

What (if any) features are you implementing?
- Better styling for app overall (UI and browser tab)

What (if anything) did you refactor?
- Update spacing/indentation
- Update "px" with "em" on CSS files
- Remove console logs

Were there any issues that arose?
- N/A

Is there anything that you need from your teammate?
- N/A

Any other comments, questions, or concerns?
- Resolves #34 
- Resolves #35  
- Resolves #36 
- Resolves #37

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# 🌳Treezy
+# 🌳 Treezy
```
Comment (BrandonAinsworth) [inline README.md:1]: Good catch, this would've been very bad

Comment (Baskanbetul) [inline README.md:1]: Such a pretty name, 🐢

**Review by Baskanbetul — APPROVED** [review]
you have good catch for annotate 🫴🏻

File: src/components/DetailsContainer/DetailsContainer.js [hunk 2]
```diff
@@ -5,92 +5,90 @@ import Header from '../Header/Header';
...
 
-    useEffect(() => {
-      console.log('match',match)
-      getSingleTree(match)
-        .then(data => {
-          setMyTree(data[0])
-        })
-        console.log('myTree',myTree)
-    },[])
+  useEffect(() => {
```
Comment (BrandonAinsworth) [inline src/components/DetailsContainer/DetailsContainer.js:13]: Excellent job getting the fetch running on page load, no more undefined data on refresh! 🥳

**Review by BrandonAinsworth — APPROVED** [review]

**Review by rachel-bock — APPROVED** [review]
Thank you for making the changes to the indentation and the tab of the app.

**Pull request merged**